portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("Hello Nitish. \t Just wanted to tell you, Ur Special!!");
```

Comment:
nit: `Ur` is intentional? or it had to be `You are`?

Reply:
It is intentional..
there is a popular youtube creater `coolman coffeedan`.. he used to say that.. so I included it :sweat_smile:

Reply:
lol! that's fine then!